src/transformers/models/whisper/generation_whisper.py

```diff
@@ -474,7 +482,7 @@ def generate(
         # 2. set global generate variables
         input_stride = self.model.encoder.conv1.stride[0] * self.model.encoder.conv2.stride[0]
         num_segment_frames = input_stride * self.config.max_source_positions
-        total_input_frames = self._retrieve_total_input_frames(
+        batch_size, total_input_frames = self._retrieve_total_input_frames(
```

Comment:
No longer need to define `batch_size` on L611 after defining here: https://github.com/huggingface/transformers/blob/1ed2880e41fea1b037313bb7e3102df213a4168e/src/transformers/models/whisper/generation_whisper.py#L611